CONTRIBUTING.rst

```diff
@@ -0,0 +1,75 @@
+Contributing to this repository
+===============================
+
+|:smile:| |:heart_eyes:| First off, thanks for taking the time to contribute! With your help, we’re making developers' lives easier. |:heart_eyes:| |:smile:|
+
+The following is a set of tips and guidelines that will help you improve the Aiven developer portal.
+
+What should I know before I contribute?
+---------------------------------------
+
+**What is devportal?**
+Devportal is Aiven’s developer portal. It’s goal is to provide a comprehensive set of help articles aimed at developers using Aiven’s products. 
+
+**How does devportal work?**
+It is Python-based, with content in ReStructuredText (rst) and rendered by Sphinx.
+
+**What is the content model?**
+Devportal contains the following content types:
+- *Concept* - Conceptual content helps people understand a feature or topic by providing a clear, high-level overview, explanation of how the feature or topic can help them on their journey, and context like use cases or examples. 
+- *Task* - Task content helps people complete a task from start to finish while they're using Aiven’s products.
+- *Reference* - Referential content provides detailed information that peoplr need while they're actively using Aiven's products.
+- *Tutorial* - Tutorials help people learn about products, and solve real world problems by guiding them through the entire workflow to complete a task.
+
+**What is the structure of devportal?**
+You can find the high-level content structure for the devportal in the following Slab article. 
+
+
+How can I contribute?
+---------------------
+
+**Reporting bugs, and suggesting enhancements**
+If you notice something is off (anything - from typos to incorrect instructions) or missing, consider opening an issue. 
+.. tip::
+    Before you open an issue, check the article request backlog, and all other open issues for the devportal. 
```

Comment:
I think we can only ask people to check the issues list since the article request backlog isn't publicly available. Could you also say what you're asking them to check for? (I'm assuming duplicate issues/requests, but I could be missing something else here)